<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Asen-PC" name="演练培训_培训管理_培训课程" tests="13" failures="7" timestamp="31 Dec 2015 08:08:56 GMT" time="64.521" errors="0">
  <testcase name="add" time="0.076" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.util.SeleniumUtil.isDisplayed(SeleniumUtil.java:462)
	at com.auto.business.drilltraining.Trainingcoursebusiness.add(Trainingcoursebusiness.java:43)
	at com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test.add(Drilltraining_Trainingcourse_test.java:40)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- add -->
  <testcase name="add" time="0.037" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.util.SeleniumUtil.isDisplayed(SeleniumUtil.java:462)
	at com.auto.business.drilltraining.Trainingcoursebusiness.add(Trainingcoursebusiness.java:43)
	at com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test.add(Drilltraining_Trainingcourse_test.java:40)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- add -->
  <testcase name="add" time="15.919" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test"/>
  <testcase name="add" time="2.776" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test"/>
  <testcase name="add" time="4.058" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test"/>
  <testcase name="add" time="3.93" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test"/>
  <testcase name="add" time="3.87" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test"/>
  <testcase name="delete" time="1.052" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test">
    <failure type="java.lang.AssertionError" message="expected [确认要删除培训课程应急管理知识吗？] but found [确认要删除培训课程系长吗？]">
      <![CDATA[java.lang.AssertionError: expected [确认要删除培训课程应急管理知识吗？] but found [确认要删除培训课程系长吗？]
	at com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test.delete(Drilltraining_Trainingcourse_test.java:78)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- delete -->
  <testcase name="delete" time="1.133" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test">
    <failure type="java.lang.AssertionError" message="expected [确认要删除培训课程课程名称12吗？] but found [确认要删除培训课程系长吗？]">
      <![CDATA[java.lang.AssertionError: expected [确认要删除培训课程课程名称12吗？] but found [确认要删除培训课程系长吗？]
	at com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test.delete(Drilltraining_Trainingcourse_test.java:78)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- delete -->
  <testcase name="query" time="1.816" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test"/>
  <testcase name="query" time="1.726" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test.query(Drilltraining_Trainingcourse_test.java:59)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- query -->
  <testcase name="query" time="1.641" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test.query(Drilltraining_Trainingcourse_test.java:59)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- query -->
  <testcase name="query" time="1.474" classname="com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.auto.cases.drilltraining.Drilltraining_Trainingcourse_test.query(Drilltraining_Trainingcourse_test.java:59)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- query -->
</testsuite> <!-- 演练培训_培训管理_培训课程 -->
